Pushing Popup and Sketch changes

YajurG opened the pull request:
## Description
Pushing changes to Popup and Sketch changes upstream

### Why

Resolves [#302, #307]

Note: While the above issues are closed, there was some additional feedback given and changes requested from the Accessibility team, which are addressed in this PR even though the linked issues are resolved.

### What

Changed 'Close Popup' buttons to use `<Button>` component instead of `<TouchableHighlight>`. 

## Screenshots

No visual difference, just a change in the UIA tree to remove extra Text component which was having unintended Accessibility problems.

chrisglein commented:
Both the linked issues are closed:
- https://github.com/microsoft/react-native-gallery/issues/302
- https://github.com/microsoft/react-native-gallery/issues/307
(from your PR 2 weeks ago)

What's the addition here?

YajurG commented:
> What's the addition here?

Made some changes addressing feedback from the accessibility team in order to get their sign off. This is the most up to date version of the package that we sent to them yesterday.

AgneLukoseviciute commented on windows/rngallery/Package.appxmanifest:
```diff
@@ -13,8 +13,8 @@
     Name="Microsoft.ReactNativeGallery"
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
-    Version="1.0.18.0" />
+    Publisher="CN=Microsoft Corporation (Internal Use Only), O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
```
I don't think we want to merge this change into the repo, only needed as a local change when signing a new package through the process you used.

AgneLukoseviciute commented on windows/rngallery/rngallery.vcxproj:
```diff
@@ -49,6 +49,15 @@
   <PropertyGroup Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <CharacterSet>Unicode</CharacterSet>
+    <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
```
same for this, don't think we want this merged into the repo.

AgneLukoseviciute reviewed (commented):
This seems to be missing the SketchCanvas changes to fix the lack of UIA control types: 
![image](https://user-images.githubusercontent.com/41223743/234961796-8d65fe27-6b27-48b0-bffd-9f9e537defeb.png)

was that included in the original SketchCanvas PR?

YajurG commented:
> This seems to be missing the SketchCanvas changes to fix the lack of UIA control types

I think those changes have already been pushed upstream. I ran the build locally before pushing and the sketch page was updated with the UIA types as button. Did you run into this just now?
